sklbench/benchmarks/sklearn_estimator.py

```diff
@@ -345,23 +347,27 @@ def ndarray_function(x, y):
                     x[i * batch_size : (i + 1) * batch_size],
                     y[i * batch_size : (i + 1) * batch_size],
                 )
+            estimator_instance._onedal_finalize_fit()

```

Comment:
Estimator instance is generic sklearn-like estimator without necessary finalize_fit
```suggestion
            if hasattr(estimator_instance, "_onedal_finalize_fit"):
                estimator_instance._onedal_finalize_fit()
```

Reply:
but if we are here it means that estimator has partial_fit method and it implies that it has finalize_fit method